src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let accumulator = initialValue;
+    let startValue = 0;
+
+    if ([...arguments].length < 2) {
+      accumulator = this[0];
+      startValue++;
```

Comment:
It would be better to assign a value (startIndex = 1) instead of just adding.

Reply:
done 👌